ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -95,3 +95,60 @@ keys:
                         type: int
                         convert_types:
                           - str
+                  police:
+                    type: dict
+                    keys:
+                      rate:
+                        type: dict
+                        keys:
+                          lower_rate:
+                            type: int
+                            convert_types:
+                              - str
+                            description: |-
+                              Specify lower rate.
+                              Lower rate range in kbps - <8-200000000>.
+                          rate_unit:
+                            type: str
+                            valid_values:
+                              - "bps"
+                              - "kbps"
+                              - "mbps"
+                              - "pps"
+                            default: "bps"
+                          lower_rate_burst_size:
+                            type: int
+                            convert_types:
+                              - str
+                            min: 256
```

Comment:
this min and max range in bytes so if user give 10 bytes then its not valid but if user give 10 kbytes then its valid.
```
wan1-dc1(config-pmap-c-quality-of-service-teat1-Test1)#police rate 10 kbps burst-size 10 bytes 
% Invalid burst size for lower rate
% Burst range <256-128000000> bytes
wan1-dc1(config-pmap-c-quality-of-service-teat1-Test1)#police rate 10 kbps burst-size 10 kbytes 
wan1-dc1(config-pmap-c-quality-of-service-teat1-Test1)#
```

Reply:
Fixed, thanks